tools/dreport.d/openpower.d/opdreport

```diff
@@ -225,12 +253,18 @@ while [[ $# -gt 1 ]]; do
         -i|--dumpid)
             dump_id=$2
             shift 2 ;;
-        -p|--path)
-            content_path=$2
-            shift 2 ;;
         -s|--size)
             dump_size=$2
             shift 2 ;;
+        -f|--failingunit)
+            failing_unit=$2
+            shift 2 ;;
+        -e|--eid)
```

Comment:
Please the new options in help i.e LN11 and README as well.

Reply:
updated help.. there is no mention of parameters in README so no change is needed there